tests/index.spec.js

```diff
@@ -26,5 +26,12 @@ describe('JWT', () => {
 
       expect(decoded).toEqual(body);
     });
+    it(`can successfully decode a body without knowing the secret for algorithm ${algorithm}`, () => {
```

Comment:
Could you please add a blank line between test cases? 

Just so it matches all other tests files.

Reply:
Fixed in c2906c6e62b3d9aa9cc1025cc57afc0a1fed2191